routes/agencies.js

```diff
@@ -2,6 +2,37 @@ const express = require('express');
 const router = express.Router();
 const path = require('path');
 var HttpStatus = require('http-status-codes');
+var _ = require('lodash');
+
+function bedCalc(agency) {
+  if (agency.total_beds_available != undefined && agency.total_beds_available != null) {
```

Comment:
I think you could get away with
```
if (agency.total_beds_available) 
```
b/c I don't think total_beds_available would every be undefined